regexes/device/mobiles.yml

```
@@ -7787,6 +7787,9 @@ Logicom:
     - regex: 'Le Lift'
       model: 'Le Lift'
       device: 'smartphone'
+    - regex: 'Le Hola (FR)[);/ ]'
+      model: 'Le Hola $1'
+      device: 'smartphone'
```

Guess there is actually no need to have the `FR` in a match...

```suggestion
    - regex: 'Le Hola FR[);/ ]'
      model: 'Le Hola FR'
      device: 'smartphone'
```